ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -345,7 +351,7 @@ struct ZipBuilder {
   /// - Throws: Various FileManager errors in case the copying fails, or an error if the framework
   //            doesn't exist in `frameworkLocations`.
   @discardableResult
-  func copyFrameworks(fromPods installedPods: [CocoaPodUtils.PodInfo],
+  func copyFrameworks(fromPods installedPods: [String],
```

Comment:
Only need the names

Reply:
Can you please change the comment to `Names of all the Pods installed...` or something like that?

Reply:
Done.